micropip/_compat/_compat_in_pyodide.py

```diff
@@ -5,9 +5,21 @@
 if TYPE_CHECKING:
     pass
 
+import pyodide
+from packaging.version import parse
 from pyodide._package_loader import get_dynlibs
 from pyodide.ffi import IN_BROWSER, to_js
-from pyodide.http import pyfetch
+
+if parse(pyodide.__version__) > parse("0.27"):
+    from pyodide.http import HttpStatusError, pyfetch
+else:
+
+    class HttpStatusError(Exception):  # type: ignore [no-redef]
+        """we just want this to be defined, it is never going to be raised"""
+
+        pass
```

Comment:
I removed Pyodide versions < 0.27 from CI. I am fine with removing these conditions and drop backward compatibility for versions < 0.27 @hoodmane WDYT?